### ISSUE TITLE
[firestore-bigquery-export] Cloud Function location is always configured as us-central1

### ISSUE THREAD

**Issue by mikeymace** [firebase/extensions#1844]
- Extension name: firebase/firestore-bigquery-export
- Extension version: 0.1.40
- Configuration values (redact info where appropriate):
  - LOCATION=europe-west2
  - DATASET_LOCATION=europe-west2
  - COLLECTION_PATH=accounts
  - DATASET_ID=firestore
  - TABLE_ID=accounts
  - USE_COLLECTION_GROUP_QUERY=No
  - DO_BACKFILL=Yes

I'm trying to deploy the firebase-bigquery-export extension using the following CLI command and the config above (located in 
accounts.env):
`firebase deploy --only extensions --project=project-X`

However, the Cloud Function location is always configured as us-central1.

I've manually configured the extension through the Firebase console and I am able to correctly configure it as europe-west2 so I know that it should be possible to configure the location outside of the default us-central1.

I am using a Cloud Firestore database.

**Comment by marknesh** [comment]
This issue has already been solved in #1750 
You can reconfigure the cloud function location as follows:
```
LOCATION=europe-west2
firebaseextensions.v1beta.function/location=europe-west2
firebaseextensions.v1beta.v2function/location=europe-west2
```

**Comment by cabljac** [comment]
Yup, thanks @marknesh this is the correct solution, here's an additional thread with info:

https://github.com/firebase/extensions/issues/1651#issuecomment-1767943894

**Comment by Naihell** [comment]
Hi everyone! I got a similar issue... I can perfectly choose CF location, but I always face this error:

![image](https://github.com/firebase/extensions/assets/17045043/1abd8860-fac7-4c73-b4fa-3157bbd4f17b)

I would like to deploy my CFs at south america location, I can see that they are properly configured there, but for some reason queues/* is at us-central1.

I've ran a test using my CFs at us-central1 and everything worked, so I guess is something between my south american settings and something adding this us-central1 setting.

My desired location: southamerica-east1
